Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityGestureController.cs

```diff
@@ -84,15 +85,41 @@ public void SetupFromInteractionSource(InteractionSourceState interactionSourceS
         /// Load the Interaction mappings for this controller from the configured Controller Mapping profile
         /// </summary>
         /// <param name="mappings">Configured mappings from a controller mapping profile</param>
-        private void SetupFromMapping(InteractionMapping[] mappings)
+        private void SetupFromMapping(IInteractionMapping[] mappings)
         {
             for (uint i = 0; i < mappings.Length; i++)
             {
                 // Add interaction for Mapping
-                Interactions.Add(mappings[i].InputType, new InteractionMapping(i, mappings[i].AxisType, mappings[i].InputType, mappings[i].InputAction));
+                switch (mappings[i].AxisType)
```

Comment:
not sure this should be populated yet. Gesture controllers need some more thinking.

Reply:
Scaffold, but I shall remove the copied code